.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
 -   repo: https://github.com/pycqa/isort
-    rev: 5.10.1
+    rev: 5.12.0
```

Comment:
For reference: https://levelup.gitconnected.com/fix-runtimeerror-poetry-isort-5db7c67b60ff